resources/test/tests/functional/api-tests-1.html

```diff
@@ -193,7 +193,7 @@ <h1>Sample HTML5 API Tests</h1>
     async_test("test should timeout (fail) with a custom set timeout value of 1 second",
                {timeout:1000}).step(function(){});
 
-    async_test("async test that is never started, should have status Not Run", {timeout:1000});
+    async_test("async test that is never started, should have status Not Run");
```

Comment:
This test will need to be updated (probably by removing it), since it's specifically about timing out after one second. There's a failing travisci job because of this change.

Reply:
@jdm, thanks for your remark. You're right, I'll remove it and other similar tests about timing out.